src/main/modules/helmet/index.ts

```diff
@@ -40,26 +36,15 @@ export class Helmet {
       helmet({
         contentSecurityPolicy: {
           directives: {
-            connectSrc: [
-              self,
-              azureMediaPlayer,
-              azureMediaServices,
-              azureMediaServicesKeyDelivery,
-              dynatraceDomain,
-              mkPlayer,
-              bitmovinLicensing,
-              mkStreaming,
```

Comment:
I think removing this you get lot of cors errors on portal player

Reply:
i think formatting wise it looks like those are being removed but actually. These are still present:
self, dynatraceDomain, mkPlayer, bitmovinLicensing, mkStreaming, mkLicense.

So only these are removed:
azureMediaPlayer,
azureMediaServices,
azureMediaServicesKeyDelivery,